src/components/ShortyString/ShortyString.tsx

```diff
@@ -20,19 +23,29 @@ type Props = {
 export default function ShortyString({
     value = '',
     limit = 200,
+    strict = false,
     displayLength = true,
     render = (v: string) => v,
     onToggle,
-    expandLabel = 'Show more',
-    collapseLabel = 'Show less',
+    expandLabel = i18n('default_expand_label'),
+    collapseLabel = i18n('default_collapse_label'),
 }: Props) {
     const [expanded, setExpanded] = React.useState(false);
-    const hasToggle = value.length > limit;
-    const length =
-        displayLength && !expanded ? `(${value.length} symbols)` : undefined;
 
-    const text = expanded || value.length <= limit ? value : value.slice(0, limit - 4) + '\u00a0...';
-    const label = expanded ? collapseLabel : expandLabel;
+    const toggleLabelAction = expanded ? collapseLabel : expandLabel;
+    const toggleLabelSymbolsCount = displayLength && !expanded
+        ? ` (${value.length} symbols)`
```

Comment:
Since you add i18n translations, it will be great to translate “symbols” as well

Reply:
and plural form also

Reply:
missed it, thanks